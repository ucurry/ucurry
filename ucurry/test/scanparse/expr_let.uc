-- def -> expr -> let
let int x = 1 in x;
let int x = 1, bool v = 2 in if v then x else x;
