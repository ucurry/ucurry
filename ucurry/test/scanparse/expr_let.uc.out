Printing Roundtrip
pretty printer not implemented for this AST node
;
pretty printer not implemented for this AST node
;
